store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -9154,6 +9167,13 @@ public void updateFileSharedWithMe(Mailbox mbox, OperationContext octxt, String
         }
     }
 
+    public String returnData(String paramA, String paramB) {
```

Comment:
I did not mean to create such a method for a ternary operator :)

Reply:
reverted back to ternary operator